docs/1.0. Certificate Provisioning.md

```diff
@@ -179,7 +180,9 @@ It is RECOMMENDED that the EST Server signs the TLS Certificate using an RSA key
 
 ## EST Client
 
-The EST Client manufacturer SHOULD issue a unique TLS Client Certificate for every device. It is RECOMMENDED that this certificate be valid for a maximum of 10 years. The Root CA used to sign it, is RECOMMENDED to be valid for a maximum of 20 years.
+The EST Client manufacturer SHOULD issue a unique TLS Client Certificate for every device. It is RECOMMENDED that the client certificate be valid for a maximum of 10 years, however the certificate SHOULD NOT constrain the device's lifetime. The Root CA used to sign the client certificate, is RECOMMENDED to be valid for a maximum of 20 years.
+
+The device manufacturer MUST make the public key of the CA available to the customer network  administrators. It is RECOMMENDED that manufacturers maintain a public Certificate Revocation List (CRL).
```

Comment:
They should potentially maintain an OCSP server and use OCSP stapling too, but this is unlikely to work in an environment without Internet access.